packages/components/src/menu-group/index.js

```diff
@@ -18,14 +17,10 @@ import { NavigableMenu } from '../navigable-container';
 export function MenuGroup( {
 	children,
 	className = '',
-	filterName,
 	instanceId,
 	label,
 } ) {
-	const childrenArray = Children.toArray( children );
-	const menuItems = filterName ?
-		applyFilters( filterName, childrenArray ) :
-		childrenArray;
+	const menuItems = Children.toArray( children );
```

Comment:
Can we now skip `toArray` and go straight for:

```js
if ( ! Children.count( children ) ) return null;

return (
  …
  { children }
```

Reply:
Right, we probably can refactor further 👍